packages/circuit-ui/components/RadioButton/RadioButton.tsx

```diff
@@ -113,6 +126,8 @@ const RadioButtonLabel = styled('label')<LabelElProps>(
   labelBaseStyles,
   labelInvalidStyles,
 );
+const RadioButtonHelperText =
+  styled('span')<LabelElProps>(helperTextBaseStyles);
```

Comment:
You can replace this custom styled element with the `FieldDescription` component from the `FieldAtoms` folder. This would ensure consistent styling and would resolve the disabled styling issue.